index.html

```diff
@@ -44,9 +44,49 @@
         </div>
       </div>
     </header>
+  
     
 
     <!-- Block banner -->
-    <main class="banner"></main>
+  <main class="banner">
```

Comment:
the main tag is not only for the banner it's for the whole site's content

Reply:
create a banner section inside the `main` tag (use the `section` tag for that) https://doka.guide/html/section/